app/styles/components/_hero.scss

```diff
@@ -1,4 +1,5 @@
 $hero-max-width: 600px;
+$team-margin-small: 10px;
```

Comment:
The naming doesn´t seem right to me. Usually in `SUIT CSS`, the variables names start with the `component` name. You can read more about the naming convention and component scope [here](https://github.com/suitcss/suit/blob/master/doc/components.md#component-scope)